.gitignore

```diff
@@ -0,0 +1,5 @@
+*.hex
+*.mikro-e
+*.map
+*.a
+obj_mikro-e/**/*.*
```

Comment:
it might be good to add .gitignore however for consistency need to be done at other clicker-apps repos as well. Any other opinions?